UPGRADES.md

```diff
@@ -1,2 +1,6 @@
 # Deprecation
+
+- Named template `alfresco-activiti.activiti-5113.elasticsearch` is now just
+  returning the value of the `ACTIVITI_ES_REST_CLIENT_ADDRESS` entry of the
+  `environment` map. It will be retired completely in the next major release.
```

Comment:
```suggestion
  `environment` map.
```

every deprecation is going to be removed on the next major release, maybe have an introduction paragraph which explain this before the list

we should also add which chart is this otherwise is not clear